.thoth.yaml

```diff
@@ -0,0 +1,26 @@
+host: khemenu.thoth-station.ninja
+tls_verify: true
+requirements_format: pipenv
+
+runtime_environments:
+  - name: 'rhel:8'
+    operating_system:
+      name: rhel
+      version: "8.0"
+    python_version: "3.6"
+    recommendation_type: latest
+    limit_latest_versions: 5
+
```

Comment:
we dont need this here as well, right?

Reply:
We can remove these

Reply:
Yes, `limit_latest_versions: 5` is no longer supported. If you spot it used somewhere, please remove it from `.thoth.yaml`.